setup.md

```diff
@@ -70,30 +70,33 @@ If you run into any difficulties, please request help before the workshop begins
         press enter to prepend Anaconda to your `PATH` (this makes the Anaconda
         distribution the default Python).
 
-> ## conda version
->
-> If you already have anaconda installed at your computer, make sure you have an up-to-date version of conda running.
-> See [these instructions](https://docs.anaconda.com/anaconda/install/update-version/) for updating conda.
->
-{: .callout}
-
 ## Installing the required packages
 
-Open a terminal and type the command (note that installing tensorflow causes keras to
-be installed too):
+[Conda](https://docs.conda.io/projects/conda/en/latest/) is the package management system associated with Anaconda and runs on Windows, macOS and Linux. 
+If you have installed Anaconda successfully, Conda should be already available in your system and work regardless of the operating system.
+Make sure you have an up-to-date version of Conda running.
+See [these instructions](https://docs.anaconda.com/anaconda/install/update-version/) for updating Conda. 
+{: .callout}
+
+Open a terminal and type the command:
 ~~~
-$ conda install tensorflow seaborn scikit-learn pandas
+conda install tensorflow seaborn scikit-learn pandas
 ~~~
-{: .language-bash}
+{: .source}  
+Note that installing Tensorflow causes Keras to be installed too.
+
 
 ### Troubleshooting for Windows
 It is possible that Windows users will run into version conflicts. If you are on Windows and get 
 errors running the command, you can try installing the packages using pip:
 
 ~~~
-$ pip install tensorflow>=2.5 seaborn scikit-learn pandas
+pip install tensorflow>=2.5 seaborn scikit-learn pandas
 ~~~
-{: .language-bash}
+{: .source}
```

Comment:
I am confused by this line. When did this API change come into our jekyll dependencies?

Reply:
I admit confusion as well. The intention was to have the code lines treated as instructions for a generic command-line interpreter rather than for a Bash terminal.  

Perhaps this confusion, if applicable, is also a reason why the current Code Check fails, although the corresponding error explanation is empty:

![image](https://user-images.githubusercontent.com/21219611/152650786-2928a3ff-75cf-4fe1-a8cd-5234f16a5431.png)

Reply:
The code check error is probably resolved if you merge master into this branch, because it looks like the error that was fixed in PR #192

Reply:
Anyway, the code checker only checks the Python code. 
Jekyll does render the blocks with `{. source}` correctly so I think that's fine.